packages/loopring_v3/contracts/aux/access/IBlockReceiver.sol

```diff
@@ -11,10 +11,8 @@ import "../../amm/libamm/AmmData.sol";
 abstract contract IBlockReceiver
 {
     function beforeBlockSubmission(
-        ExchangeData.Block memory _block,
-        bytes              memory data,
-        uint                      txIdx,
-        uint                      numTxs
+        bytes              calldata txsData,
+        bytes              calldata data
```

Comment:
We probably need to rename data to something more explicit and add a comment for this method.